inc/vcf/error.hpp

```diff
@@ -269,8 +269,15 @@ namespace ebi
         using BodySectionError::BodySectionError;
```

Comment:
not sure about this, a `DuplicationError` is also a body section error, but it doesn't refer to any specific column. I guess the Fixer class or its methods could define named constants for clarity